src/Geta.Optimizely.Sitemaps/Controllers/GetaSitemapController.cs

```diff
@@ -57,7 +57,8 @@ public ActionResult Index()
             return RealtimeSitemapData(sitemapData);
         }
 
-        return SitemapData(sitemapData);
+        // returned cached version from schedule job instead of running through generator.
+        return FileContentResult(sitemapData);
```

Comment:
shouldn't this be `return FileContentResult(sitemapData.Data)` instead?

`sitemapData` is an object coming from DDS, containing stuff like RootPageID etc

Reply:
ah, my bad, I see now that `.Data` is accessed inside `FileContentResult` method.